Dockerfile

```diff
@@ -14,6 +14,18 @@ ENV CGO_ENABLED 0
 RUN go build -o /assets/in ./cmd/in
 RUN go build -o /assets/out ./cmd/out
 RUN go build -o /assets/check ./cmd/check
+RUN set -e; for pkg in $(go list ./... | grep -v "acceptance"); do \
+    go test -o "/tests/$(basename $pkg).test" -c $pkg; \
+    done
+
+FROM builder AS tests
+COPY --from=builder /tests /go-tests
+COPY --from=builder /src/bosh/fixtures /go-tests/fixtures
+WORKDIR /go-tests
+RUN set -e; for test in /go-tests/*.test; do \
```

Comment:
What is this second stop doing? It's iterating over the output from the `go test` commands that were run above?

Reply:
Yep, this actually runs the compiled tests from above